moped-editor/src/views/staff/StaffListViewCustomComponents.js

```diff
@@ -49,12 +49,15 @@ export const EditUserButton = ({ id }) => (
 const mugUserFilter = (user) => user.is_user_group_member && !user.is_deleted;
 
 /**
- * Filter function to identify all active users who are not non-login users
+ * Filter function to identify all active users who are not non-login users and
+ * do not have a `+` in their email address
  * @param {object} user - a `moped_user` object
  * @returns user
  */
 const allUserFilter = (user) =>
-  !user.is_deleted && !user.roles.includes("non-login-user");
+  !user.is_deleted &&
+  !user.roles.includes("non-login-user") &&
+  user.email.indexOf("+") < 0;
```

Comment:
this excludes our test accounts e.g. "transportation.data+mopedadmin@austintexas.gov"